src/fielddef.ts

```diff
@@ -485,9 +498,34 @@ export function resetTitleFormatter() {
 }
 
 export function title(fieldDef: FieldDefBase<string>, config: Config) {
+  if (isFieldDef(fieldDef)) {
+    if (isPositionFieldDef(fieldDef) && fieldDef.axis && fieldDef.axis.title) {
```

Comment:
This logic will ignore if user intentionally specify title to be '' or false.

For axis/legend/header, we want to respect if users want to disable the title.
However, for tooltip, we should ignore the disabling. 

Thus, let's one more object params:   `, {allowDisabling}: {allowDisabling: boolean}`.  
Note that this is more readable than boolean param `allowDisabling: boolean` as the method will be called like `title(fieldDef, config, {allowDisabling: true}` instead having a vague boolean param  `title(fieldDef, config, true)`